exasol/transformers_extension_wrapper.py

```diff
@@ -95,21 +80,21 @@ def deploy_language_container(conf: Secrets,
 
     deployer = TeLanguageContainerDeployer.create(
         dsn=get_external_host(conf),
-        db_user=conf.USER,
-        db_password=conf.PASSWORD,
-        bucketfs_name=conf.BUCKETFS_SERVICE,
-        bucketfs_host=conf.get("BUCKETFS_HOST_NAME", conf.EXTERNAL_HOST_NAME),
-        bucketfs_port=int(conf.BUCKETFS_PORT),
-        bucketfs_user=conf.BUCKETFS_USER,
-        bucketfs_password=conf.BUCKETFS_PASSWORD,
-        bucketfs_use_https=str_to_bool(conf, "BUCKETFS_ENCRYPTION", True),
-        bucket=conf.BUCKETFS_BUCKET,
+        db_user=conf.get(CKey.db_user),
+        db_password=conf.get(CKey.db_password),
+        bucketfs_name=conf.get(CKey.bfs_service),
+        bucketfs_host=conf.get(CKey.bfs_host_name, conf.get(CKey.db_host_name)),
+        bucketfs_port=int(str(conf.get(CKey.bfs_port))),
+        bucketfs_user=conf.get(CKey.bfs_user),
+        bucketfs_password=conf.get(CKey.bfs_password),
+        bucketfs_use_https=str_to_bool(conf, CKey.bfs_encryption, True),
```

Comment:
if the member names are all the same as the setting names this could be simplified with a loop. e.g.

```python
settings = ['db_user', 'db_password', ...]
for setting in settings:
    value = getattr(conf, setting)
    setattr(self, setting, value)
```

Reply:
Well, they are not the same